content/basics/use-ably.textile

```diff
@@ -0,0 +1,80 @@
+
+* maintain a persistent connection to Ably 
+* attach to one or more channels, and publish and subscribe to messages to them
+* register their presence on a channel, or listen for others present in realtime
+* publish at very high message rates, or with the lowest possible realtime latencies
+
+h2(#rest-api). REST HTTP API
+
+Interacting with the "REST HTTP API":/api/rest-api directly is fully supported. However, Ably recommends using the REST interface of an SDK where possible, as they provide additional features that improve performance and resilience that the REST HTTP API can't deliver on its own. This includes automatic re-routing around network problems by using alternative data centers. 
+
+h2(#protocol-adapters). Protocol adapters
+
+"Protocol adapters":/protocols are an alternative method of using Ably and provide interoperability between Ably's protocol and other popular protocols, such as MQTT. They are less feature-rich than Ably SDKs, however they require fewer resources in terms of memory and network overhead to run. Protocol adapters can also be used in applications built using platforms that don't have an available Ably SDK.
+
+The following protocol adapters are available:
+
+* "MQTT":#mqtt
+* "Ably SSE and EventSource API":#sse
+* "Pusher and PubNub":#pusher-pubnub
+
+h3(#mqtt). MQTT
+
+The Ably "MQTT protocol adapter":/protocols/mqtt translates between the "Message Queuing Telemetry Transport (MQTT)":/https://mqtt.org and Ably’s own protocol. MQTT is a lightweight messaging protocol optimized for high-latency and unreliable networks. It is often used in remote devices with small footprints. MQTT libraries are available for most 
+internet-enabled devices.
+
+You can use the MQTT protocol adapter with your MQTT library when:
+
+* an Ably SDK isn't available for your target platform
+* bandwidth is limited and you want to keep network traffic to a minimum
+* you want to avoid any type of vendor lock-in by using open protocols
+
+h3(#sse). Ably SSE and EventSource API
+
+The "Ably SSE (Server Sent Events) and raw HTTP streaming API":/protocols/sse provides a way to get a realtime stream of events from Ably when using a full Ably SDK is impractical. HTTP streaming enables a client's requests to be held by a server, enabling it to push data to the client without further requests. Note that it isn't possible to publish using SSE. It is only possible to subscribe to messages and events.
+
+You can use the SEE protocol adapter when:
+
+* an Ably SDK isn't available for your target platform
+* you have stringent memory restrictions
+* you only need to subscribe to events on channels, not publish them
+
+h3(#pusher-pubnub). Pusher and PubNub
+
+Ably provides protocol adapters for "Pusher":/tutorials/pusher-adapter and "PubNub":/tutorials/pubnub-adapter. These protocol adapters only require you to change your API key. This enables you to migrate from Pusher or PubNub incrementally, or all at once.
```

Comment:
If my understanding of 'These protocol adapters only require you to change your API key.' is correct as 'All you need to change in your existing code is the API key', then that's not quite true as they also need to update the origin to 'PROVIDER.ably.io'. The API key of course also needs to be an Ably API key but I feel that part is implied haha.